client/src/components/RoomSettingsForm.vue

```diff
@@ -65,55 +65,54 @@
-					:disabled="!granted('configure-room.other')"
-					data-cy="input-auto-skip-outro"
-				/>
-				<v-checkbox
-					v-model="inputRoomSettings.autoSkipSegmentCategories.interaction"
-					:label="$t('room-settings.auto-skip-text-interaction')"
-					:disabled="!granted('configure-room.other')"
-					data-cy="input-auto-skip-interaction"
-				/>
-				<v-checkbox
-					v-model="inputRoomSettings.autoSkipSegmentCategories.selfpromo"
-					:label="$t('room-settings.auto-skip-text-selfpromo')"
-					:disabled="!granted('configure-room.other')"
-					data-cy="input-auto-skip-selfpromo"
-				/>
-				<v-checkbox
-					v-model="inputRoomSettings.autoSkipSegmentCategories.music_offtopic"
-					:label="$t('room-settings.auto-skip-text-music_offtopic')"
-					:disabled="!granted('configure-room.other')"
-					data-cy="input-auto-skip-music_offtopic"
-				/>
-				<v-checkbox
-					v-model="inputRoomSettings.autoSkipSegmentCategories.preview"
-					:label="$t('room-settings.auto-skip-text-preview')"
-					:disabled="!granted('configure-room.other')"
-					data-cy="input-auto-skip-preview"
-				/>
+			<div v-if="granted('configure-room.other')">
+				<v-card-text>
+					{{$t('room-settings.auto-skip-text')}}
+				</v-card-text>
+				<div class="auto-skip-categories">
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.sponsor"
+						:label="$t('room-settings.auto-skip-text-sponsor')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-sponsor"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.intro"
+						:label="$t('room-settings.auto-skip-text-intro')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-intro"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.outro"
+						:label="$t('room-settings.auto-skip-text-outro')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-outro"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.interaction"
+						:label="$t('room-settings.auto-skip-text-interaction')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-interaction"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.selfpromo"
+						:label="$t('room-settings.auto-skip-text-selfpromo')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-selfpromo"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.music_offtopic"
+						:label="$t('room-settings.auto-skip-text-music_offtopic')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-music_offtopic"
+					/>
+					<v-checkbox
+						v-model="autoSkipSegmentCategoriesObject.preview"
+						:label="$t('room-settings.auto-skip-text-preview')"
+						:disabled="!granted('configure-room.other')"
+						data-cy="input-auto-skip-preview"
+					/>
+				</div>
```

Comment:
I think something like this might work better for the UI

![image](https://github.com/dyc3/opentogethertube/assets/1808807/8717bc0e-b2b0-44ea-8c86-de2d6c6620b6)
See: https://vuetifyjs.com/en/components/selects/#chips
Or [this playground](https://play.vuetifyjs.com/#eNqNT9EKgzAM/JWQFzdwytjGQFQY+wz1oWpkhTqLrTIU/31tVdjjni53ySWXbEbVV+FDymAcCCOMNbVSME1p/gaIx5MiQZW2BGA8tV1NIslxZGKgHFc54sakjOpwV6sXl2otBSud6+mkrd8OQnMpyLI0DvdL5m4c/oQwVFU9l7YBQB/Z9Rpqapixw7yuqplmERyOkKRw2DQAlyaCzGu6zvPBK1lvoeHT5OhgsPD3affRn9PL0RWLjbqFw8XHS3AN7mjxFpyx+AIyxGr8)